Sources/SwiftBenchmarkTools/ArgParser.swift

```diff
@@ -0,0 +1,245 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the GSoC Swift Tracing open source project;
+// Based on: https://github.com/apple/swift/tree/cf53143a47278c2a465409a67376642515956777/benchmark/utils
```

Comment:
Infra based on https://github.com/apple/swift/tree/master/benchmark/utils
It's not super great but good enough to get some basic benches done